terraform/modules/gro_ingestion_service/lambdas/src/models/GroDeathRegistration.ts

```diff
@@ -0,0 +1,68 @@
+import { AttributeValue } from "@aws-sdk/client-dynamodb"
+
+export interface GroDeathRegistration {
+    RegistrationId: string,
+    RegistrationType: string,
+    RecordLockedDateTime: string,
+    RecordUpdateDateTime: string,
+    RecordUpdateReason: string,
+    DeceasedName: {
+        PersonNameTitle: string,
+        PersonGivenName: string,
+        PersonFamilyName: string,
+        PersonNameSuffix: string,
+    },
+    DeceasedNameAlias: {
+        PersonNameTitle: string,
+        PersonGivenName: string,
+        PersonFamilyName: string,
+        PersonNameSuffix: string,
+    },
+    DeceasedAliasNameType: string,
+    DeceasedMaidenName: string,
+    DeceasedGender: string,
+    DeceasedDeathDate: {
+        PersonDeathDate: string,
+        VerificationLevel: string,
+    },
+    PartialMonthOfDeath: string,
+    PartialYearOfDeath: string,
+    QualifierText: string,
+    FreeFormatDeathDate: string,
+    DeceasedBirthDate: {
+        PersonBirthDate: string,
+        VerificationLevel: string,
+    },
+    PartialMonthOfBirth: string,
+    PartialYearOfBirth: string,
+    FreeFormatBirthDate: string,
+    DeceasedAddress: {
+        Flat: string,
+        Building: string,
+        Line: string[],
+        Postcode: string,
+    },
+}
+
+
+export const mapToDynamoDbItem = (deathRecord: GroDeathRegistration, hash: string): { [key: string]: AttributeValue } => {
```

Comment:
I would rather we map to EventRecord, then we can run `marshal(eventRecord)` to get the dynamoDbItem. marshal is in `@aws-sdk/util-dynamodb`